src/main/java/com/prgrms/springJpa/domain/order/Member.java

```diff
@@ -0,0 +1,91 @@
+package com.prgrms.springJpa.domain.order;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import lombok.Getter;
+
+@Getter
+@Entity
+@Table(name = "member")
+public class Member {
+    private static final int NAME_MAX_LENGTH = 30;
+    private static final int NICK_NAME_MAX_LENGTH = 30;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "name", nullable = false, length = NAME_MAX_LENGTH)
+    private String name;
+
+    @Column(name = "nick_name", nullable = false, length = NICK_NAME_MAX_LENGTH, unique = true)
+    private String nickName;
+
+    @Column(name = "age", nullable = false)
+    private int age;
+
+    @Column(name = "address", nullable = false)
+    private String address;
+
+    @Column(name = "description")
+    private String description;
+
+    @OneToMany(mappedBy = "member")
+    private List<Order> orders = new ArrayList<>();
+
+    protected Member() {
+    }
+
+    public Member(String name, String nickName, int age, String address) {
+        this(name, nickName, age, address, null);
+    }
+
+    public Member(String name, String nickName, int age, String address, String description) {
+        this.name = name;
+        this.nickName = nickName;
+        this.age = age;
+        this.address = address;
+        this.description = description;
+        validateName(name);
+        validateNickName(nickName);
```

Comment:
```suggestion
        validateName(name);
        validateNickName(nickName);
        this.name = name;
        this.nickName = nickName;
        this.age = age;
        this.address = address;
        this.description = description;
```
별건 아니지만 이게 더 자연스러운 느낌?